awscli/customizations/s3/filegenerator.py

```diff
@@ -173,6 +173,12 @@ def list_files(self, path, dir_op):
         if not self.should_ignore_file(path):
             if not dir_op:
                 size, last_update = get_file_stat(path)
+                if last_update is EPOCH_TIME:
```

Comment:
This is...interesting.  Remind me again the issues with just raising a "could not figure out the mtime" exception and catching it here?